projects/valtimo/components/src/lib/components/form-io/form-io-builder.component.ts

```diff
@@ -27,7 +30,23 @@ export class FormioBuilderComponent implements OnInit {
   // eslint-disable-next-line @angular-eslint/no-output-native
   @Output() change: EventEmitter<any> = new EventEmitter();
 
-  constructor() {}
+  readonly currentLanguage$ = this.translateService
+    .stream('key')
+    .pipe(map(() => this.translateService.currentLang));
+
+  readonly formioOptions$: Observable<{language: string, i18n: any }> = this.currentLanguage$.pipe(
```

Comment:
I think there exists a typing exported by form.io itself?

Reply:
I have updated the the type with a default variant from form.io